lib/Field/index.js

```diff
@@ -9,7 +9,8 @@ import ExpandingTextArea from '../ExpandingTextArea';
 import EditableTextWrapper from '../EditableTextWrapper';
 
 const Field = props => {
-  const additionalClasses = cx({
+  const classNames = cx('field', {
```

Comment:
```suggestion
  const classNames = cx(`field field--${props.dir}`, {
```
as `props.dir` will always be true?